src/main/kotlin/blackjack/domain/Player.kt

```diff
@@ -0,0 +1,29 @@
+package blackjack.domain
+
+class Player(val name: String, val hand: MutableList<Card> = mutableListOf()) {
```

Comment:
```suggestion
class Player(val name: String) {
    private val _hand: MutableList<Card> = mutableListOf()
    val hand: List<Card> = _hand
```
코틀린을 안전하게 사용하려면 바깥에서 변화를 주지 않도록 하는 습관이 필요하다고 생각해요! 🤩

backingProperties를 사용하여 안에서는 가변으로 바깥에서는 불변으로 사용해볼까요?

링크
[backing-properties](https://kotlinlang.org/docs/properties.html#backing-properties)